### PR TITLE
Feature/localize setup elasticsearch command

### PR DESCRIPTION
**Pull request by phivk**
NB: this requires the following to be specified in `.env`:

```
ES_INDEX_SK=webumenia_sk
ES_INDEX_EN=webumenia_en
```

### CODE REVIEW

**Comment by phillchill** [comment]
Feedback from @igor-kamil: ideally the script would create both indexes (SK & EN) by default.
will add this functionality...

**Comment by phivk** [comment]
@igor-kamil I added the default behaviour, this PR is now ready for your review :)

File: app/Console/Commands/SetupElasticsearch.php [hunk 1]
```diff
@@ -37,447 +37,129 @@ public function __construct()
...
-                }
-              }
-            }
-          }
-        }
-        ';
+    public function test_function()
+    {
+      $this->info('Test');
+    }
```
[the remark this answers is not in the document]
Comment (phivk) [reply app/Console/Commands/SetupElasticsearch.php]: thanks, I missed this one...

**Comment by phivk** [comment]
> don't hardcode the locales
> put the json files into subfolders
> e.g. es_index_settings/{locale_code}/{type}.json

@igor-kamil great idea! I will implement and push...

**Comment by phivk** [comment]
@igor-kamil Please see the changes I've added :)

**Comment by igor-kamil** [comment]
Very nice implementation!

**Pull request merged**